R/epGRanges.R

```diff
@@ -0,0 +1,46 @@
+## Equipartition GRanges:
+## Splits a GRanges object into a specified number of parts with equal (total) width.
+## Each part is a GRanges and all parts are returned as GRangesList object. 
+
+## Alexandre Kuhn (alexandre.m.kuhn@gmail.com)
+
+epGRanges<-function(gr,partitionNb) {
```

Comment:
Would be nice if this let the user specify the desired width of each window, as an alternative to the number of windows.